server/lyft/feature/names.go

```diff
@@ -5,4 +5,5 @@ type Name string
 // list of feature names used in the code base. These must be kept in sync with any external config.
 const (
 	PlatformMode Name = "platform-mode"
+	PRMode       Name = "pr-mode"
```

Comment:
Can we just call this LegacyDeprecation or something like that, this will definitely confuse me because we have platform mode and default mode keys for example.